crates/orchestrator/src/lib.rs

```diff
@@ -636,7 +681,23 @@ where
     })?
     .post(
         "post_ready",
-        |_req, state: &mut <State as ReadState>::State| async move { state.post_ready() }.boxed(),
+        |req, state: &mut <State as ReadState>::State| {
+            async move {
+                let mut body_bytes = req.body_bytes();
```

Comment:
I think that this requires a change in `orchestrator/src/client.rs:445` to change how we form the request (since we now need to include our public key)

Reply:
Done!